File: 2021/August/Day 3/TwoSum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {

        vector<int> out;

        for(int i=0;i<size(nums);i++)
        {

            auto it = std::find(nums.begin(),nums.end(),target-nums[i]);
            if(it!=nums.end())
            {
                int index = it - nums.begin();

                if(index!=i)
                {out.insert(out.end(),i);
                out.insert(out.end(),index);
                 return out;
                }
            }
        }
        return out;
    }
};

File: 2024/Jun/twoSum.go
func twoSum(nums []int, target int) []int {
    for i, s := range nums {
	for j,p := range(nums){
        if(i!=j){
            if(s+p==target){
                out := []int{i,j}
                return out
            }
        }

    }
}
out := []int{-1,-1}
return out
}
